services/gitlab/gitlab-license.service.js

```diff
@@ -15,15 +15,16 @@ const queryParamSchema = Joi.object({
 
 const documentation = `
 <p>
-  You may use your GitLab Project Id (e.g. 13083) or your Project Path (e.g. gitlab-org/gitlab-foss )
+  You may use your GitLab Project Id (e.g. 278964) or your Project Path (e.g. gitlab-org/gitlab )
+</p>
+`
+
+const customDocumentation = `
+<p>
+  Note that only network-accessible jihulab.com and other self-hosted GitLab instances are supported.
```

Comment:
Let's adjust this a bit:

```suggestion
  Note that only internet-accessible GitLab instances are supported, for example https://jihulab.com, https://gitlab.gnome.org, or https://gitlab.com/.
```